### PR TITLE
Use a different check mark character on Windows Terminal

### LINKED ISSUE #1: Can't tell if an item is selected or not on Windows Terminal
The tick appears purple whether or not the branch is selected.

This appears to be due to: https://github.com/microsoft/terminal/issues/13110

See also:
- https://github.com/microsoft/terminal/issues/15592
- https://github.com/console-rs/dialoguer/issues/264

PR forthcoming.

### PR DESCRIPTION
**Pull request by birtles**
Fixes #1.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -31,7 +32,28 @@ fn delete_branches(branches: Vec<String>) -> i32 {
...
+//
+// Fortunately, this doesn't affect U+2713 "Check Mark" so for Windows terminal we use that
+// character instead until the emoji issue is fixed.
+const WINDOWS_TERMINAL_SYMBOLS: SymbolSet = SymbolSet {
+    completed: '\u{2713}', // '✓'
+    ..requestty::symbols::UNICODE
+};
+
+fn is_windows_terminal() -> bool {
+    std::env::var("WT_SESSION").is_ok()
```
Comment (yuki0418) [inline src/main.rs:49]: @birtles 

If the window terminal is the default terminal, `WT_SESSION` returns null.
Does this work on your PC?

https://github.com/microsoft/terminal/issues/13006#issue-1221666658

**Review by yuki0418 — COMMENTED** [review]
One comment, however code is awsome.

File: src/main.rs (hunk 1 above)
Comment (birtles) [reply src/main.rs:49]: Oh, I didn't know that. My default terminal is "Windows Terminal (preview)" and `WT_SESSION` is set for me.

I tried setting my default terminal to "Windows Terminal" (not preview) and running it but `WT_SESSION` is still set for me. Maybe it's fixed?

Comment (yuki0418) [reply src/main.rs:49]: @birtles 
Could be fixed.
I will see when same problem happen next time in different environment!!

Thanks!

**Pull request merged**